po/zh_CN.po

```diff
@@ -1154,8 +1154,8 @@ msgid "Unknown 'measure.vars' type %s at index %d of list"
 msgstr "未知'measure.vars'类型 %s，位于列表中 %d"
 
 #: fmelt.c:187
-msgid "One or more values in 'measure.vars' is invalid."
-msgstr "'measure.vars'里，一或多个数值无效"
+msgid "One or more values in 'measure.vars' is invalid; please fix by removing: %s"
+msgstr "'measure.vars'里，一或多个数值无效；请通过删除以下数值来修复：%s"
```

Comment:
typically prs like this do not attempt to translate new messages, because it makes review difficult.
for example I have no way to verify if this translation is reasonable, but I can review the code and english messages.
please remove the translations from this PR and do them in a separate PR

Reply:
Just to be clear: the current state is not ok, as the msgid was manually changed and the msgstr was kept original without a `#, fuzzy` flag, causing the outdated translation to be used without a review by translators.

Also, AFAIK, the PO files are only updated before each release. So updating msgid in this specific PR is unnecessary.

IMHO, changes to PO files needs to be undone.